tests/integration/data_sources_and_expectations/test_misconfigured_expectations.py

```diff
@@ -161,14 +161,3 @@ def test_column_min_max_mismatch_misconfiguration(batch_for_datasource) -> None:
     result = batch_for_datasource.validate(expectation)
     assert not result.success
     assert "min_value cannot be greater than max_value" in str(result.exception_info)
-
-
-@parameterize_batch_for_data_sources(
-    data_source_configs=ALL_DATA_SOURCES,
-    data=pd.DataFrame({"a": [1, 2]}),
-)
-def test_column_min_max_missing_misconfiguration(batch_for_datasource) -> None:
```

Comment:
with the validation error now being raised at the expectation level, we will no longer reach this code path (https://github.com/great-expectations/great_expectations/blob/b/gx-600/oss-expect-column-values-to-be-between-allows-both-min-max-values-to-be-none-or-empty-strings/great_expectations/expectations/metrics/column_aggregate_metrics/column_values_between_count.py#L46)